plugins/available/gitstatus.plugin.bash

```diff
@@ -0,0 +1,23 @@
+cite about-plugin
+about-plugin 'speeds up your life by using gitstatus for git status calculations. install from https://github.com/romkatv/gitstatus'
+
+
+function gitstatus_on_disable() {
+  about 'Destructor of gitstatus plugin'
+  group 'gitstatus'
+
+  unset SCM_GIT_USE_GITSTATUS
+  _command_exists gitstatus_stop && gitstatus_stop
+}
+
+[[ "$SCM_CHECK" == "true" ]] || return # No scm-check
```

Comment:
Minor comment - can you please move comments like here to the line above? Same for the command below...

Reply:
done